spec/models/member_spec.rb

```diff
@@ -89,10 +89,18 @@
           create(:member, email: nil)
         end.to_not raise_error
       end
+
+      it 'is invalid with the same email with different casing' do
+        member = build(:member, email: 'Foo@eXample.COM')
+        expect(member).to be_invalid
+      end
+
+      it 'cannot save a member with the same email with different casing' do
+        member = build(:member, email: 'Foo@eXample.COM')
+        expect { member.save }.not_to change { Member.count }
+      end
```

Comment:
I think this two specs are redundant. If a model is invalid, it won't be saved. This is expected AR behaviour, we shouldn't be testing it.